ClusterObj[] cluster_obj
bool brake_status
